### DISCUSSION TITLE
Window titlebar size

### DISCUSSION THREAD

**Discussion by silent21century** [AvaloniaUI/Avalonia#15885] [Q&A]
i'm using `ExtendClientAreaToDecorationsHint="True"` leaving only the three controls of the windowState, and that's because im looking to use a custom titlebar with those controls.
- the problem is when the window is Maximized the titlebar's height is 23pixels and when its Normal the height is 30, is there a way to make the title bar's height constant on all states?

**Comment by Pratham95VP** [comment]
I have not done something like this but you can try using styles . Create a custom Window style that targets the WindowTitleBar control and set height property into this and apply this custom style to the control.  @timunie @maxkatz6 is this the right way?

**Comment by thevortexcloud** [comment]
Seeing as you have a custom titlebar, can't you just set a fixed height?
